iceoryx_posh/test/moduletests/test_capro_service.cpp

```diff
@@ -365,146 +267,120 @@ TEST_F(ServiceDescription_test, ServiceDescriptionCtorWithServiceStringAndInstan
 
 TEST_F(ServiceDescription_test, TwoServiceDescriptionsWithAnyServiceAnyInstanceAnyEventIDsAreEqual)
 {
-    uint16_t testServiceID = iox::capro::AnyService;
-    uint16_t testEventID = iox::capro::AnyEvent;
-    uint16_t testInstanceID = iox::capro::AnyInstance;
-    ServiceDescription serviceDescription1 = ServiceDescription(testServiceID, testEventID, testInstanceID);
-    ServiceDescription serviceDescription2 = ServiceDescription(testServiceID, testEventID, testInstanceID);
+    IdString_t testService = iox::capro::AnyServiceString;
+    IdString_t testEvent = iox::capro::AnyEventString;
+    IdString_t testInstance = iox::capro::AnyInstanceString;
+    ServiceDescription serviceDescription1 = ServiceDescription(testService, testEvent, testInstance);
+    ServiceDescription serviceDescription2 = ServiceDescription(testService, testEvent, testInstance);
 
     EXPECT_TRUE(serviceDescription1 == serviceDescription2);
 }
 
 TEST_F(ServiceDescription_test,
-       ServiceDescriptionWithAnyServiceAnyInstanceAnyEventIDAndServiceDescriptionWithValidIDsAreEqual)
+       ServiceDescriptionWithAnyServiceAnyInstanceAnyEventAndServiceDescriptionWithValidStringsAreEqual)
 {
-    uint16_t testServiceID1 = 1U;
-    uint16_t testEventID1 = 2U;
-    uint16_t testInstanceID1 = 3U;
-    uint16_t testServiceID2 = iox::capro::AnyService;
-    uint16_t testEventID2 = iox::capro::AnyEvent;
-    uint16_t testInstanceID2 = iox::capro::AnyInstance;
-    ServiceDescription serviceDescription1 = ServiceDescription(testServiceID1, testEventID1, testInstanceID1);
-    ServiceDescription serviceDescription2 = ServiceDescription(testServiceID2, testEventID2, testInstanceID2);
+    IdString_t testService1 = "1";
+    IdString_t testEvent1 = "2";
+    IdString_t testInstance1 = "3";
```

Comment:
We need more `hypnotoads`, `brain slugs` and `rock stars`

Reply:
Feel free to push an additional commit :laughing: